common-ui/src/main/res/values/design-system-typograhy.xml

```diff
@@ -89,76 +90,85 @@
     <style name="Typography.DuckDuckGo">
         <item name="android:textStyle">normal</item>
         <item name="android:fontFamily">sans-serif</item>
-        <item name="android:letterSpacing">0.0</item>
     </style>
 
     <style name="Typography.DuckDuckGo.Title">
         <item name="android:textSize">32sp</item>
-        <item name="android:lineSpacingExtra">4sp</item>
+        <item name="android:lineSpacingExtra">2sp</item>
```

Comment:
The rest of typography styles follow:
-    `textSize` + `lineSpacingExtra` = `lineHeight`

However, this one doesn't. So we need to update it:
```suggestion
        <item name="android:lineSpacingExtra">4sp</item>
```
In case it's the other way around, we need to update the rest of typography styles